pymc3/distributions/continuous.py

```diff
@@ -1675,8 +1675,12 @@ def logp(value, b, kappa, mu):
             0 < kappa,
         )
 
+class Lognormal(LogNormal):
+    """deprecated, use LogNormal instead!"""
+    def __init__(self, *args, **kwargs):
+        return super().__init__(*args, **kwargs)
```

Comment:
This class needs to go below the `LogNormal`. At this point in the code the base class is not yet defined.

Also you can remove the constructor here, since it doesn't do anything. (A `pass` can be addedd if the docstring is not enough for the syntax.